torchgeo/datasets/benin_cashews.py

```diff
@@ -387,7 +387,8 @@ def _load_mask(self, transform: rasterio.Affine) -> Tensor:
             dtype=np.uint8,
         )
 
-        mask = torch.from_numpy(mask_data).long()
+        mask = torch.from_numpy(mask_data)
+        mask = mask.long()
```

Comment:
Seems like chains of PyTorch functions no longer work? Could also be related to https://github.com/pytorch/pytorch/issues/116327